delete/delete.php

```diff
@@ -1 +1,14 @@
 <?php
+$db = new PDO('mysql:host=192.168.20.20;dbname=cms_portfolio', 'root', '');
+$db-> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+?>
+<link rel="stylesheet" type="text/css" href="../add_new/add_proj.css"><link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
+<?php
+$deleteProject = 'DELETE FROM `projects` WHERE `id` = ?;';
+$query = $db->prepare($deleteProject);
+$query->execute([$_GET['id']]);
```

Comment:
no check if the $_GET['id'] is not empty

Reply:
amended in new commit